home/modules/contribute/examples/helpers/each-openapi.js

```diff
@@ -0,0 +1,164 @@
+// Format config section of openapi property definitions.
+// We pass in a data structure from an openapi Schema.
+//
+//     Table of Contents (of the tree of this Schema, in HTML, linked to below)
+//
+//     The Asciidoc template is called for each node of the tree
+//
+// see extensions-template.adoc for motivating example.
+'use strict'
+
+module.exports = function(context, ...schemas) {
+    const options = schemas.pop()
+    
+    // setup some helper variables
+    const indent = "   " 
+    const types = 
+    {
+        integer: { fallback: '0', format: parseInt },
+        number: { fallback: '0.0', format: parseFloat },
+        string: { fallback: 'string', format: (val => `"${val}"`) },
+        boolean: { fallback: 'true' },
+        array: { fallback: '', format: (val => `[${val}]`) },
+    }
+    
+    // We try to display the parameter as the best possible value,
+    // e.g.
+    //     example -> default -> fallback for the type
+    //
+    // Then, whether we used a fallback like "string" or an example like "Bob",
+    // we format it depending on its type (e.g. surrounding it in quotes etc.)
+
+    function format_type (node) {
+        if (node.type == 'object') return
```

Comment:
(minor?) My JS is rusty (some Typescript tho), but doesn't JS use triple === for boolean truthy, similar to PHP/Perl where strings are booleans, numbers and everything else

Reply:
Both work, but you are strongly advised to use `===`. The only time you should use `==` is when comparing against `null` (since that catches `undefined` too).